Source/WebCore/rendering/RenderListMarker.h

```diff
@@ -51,6 +51,20 @@ class RenderListMarker final : public RenderBox {
     LayoutUnit lineLogicalOffsetForListItem() const { return m_lineLogicalOffsetForListItem; }
     const RenderListItem* listItem() const;
 
+    struct ListMarkerTextContent {
```

Comment:
This name seems too long for a `struct` that is a member of `RenderListMarker`. Why repeat the words “text marker”?

Reply:
I'm moving this out of RenderListMarker, my latest revision does that.